littlechef/runner.py

```diff
@@ -250,6 +228,8 @@ def recipe(recipe):
     data = lib.get_node(env.host_string)
     data["run_list"] = ["recipe[{0}]".format(recipe)]
     if not __testing__:
+        if env.autodeploy_chef and not chef.chef_test():
+                deploy_chef(ask="no")
```

Comment:
Why ? __testing__ if was there before and we want to install chef client only when we have autodeploy set and it's not installed yet.